src/server/node_func.c

```diff
@@ -2388,9 +2399,12 @@ decode_Mom_list(struct attribute *patr, char *name, char *rescn, char *val)
 }
 
 
-#ifndef	WIN32
+static int	ndevices;
```

Comment:
Shouldn't you initialize ndevices and nsockets here?

Reply:
I wanted them to be zero. So static will take care of that.

Reply:
I've always initialized them explicitly. Wasn't aware that worked for every compiler.